frontend/app_flowy/packages/appflowy_editor/test/service/internal_key_event_handlers/arrow_keys_handler_test.dart

```diff
@@ -466,6 +466,58 @@ void main() async {
       ),
     );
   });
+
+  testWidgets('Presses ctrl + backspace to delete a word', (tester) async {
```

Comment:
I suggest adding more tests to cover the edge cases.

Like
A.
1. the before text: welcome to app|flowy.
2. pressing ctrl+backspace
3. the after text: welcome to app.

B.
1. the before text: |welcome to appflowy.
2. pressing ctrl+backspace
3. nothing happens.

Reply:
Sure. But in Case A. actually, we expect the after text to be 
` welcome to flowy.`